packages/api/config/lambdas.yml

```diff
@@ -4,6 +4,7 @@ sqs2sf:
   memory: 512
   source: 'node_modules/@cumulus/api/dist/'
```

Comment:
I'm thinking this one might need a dlq?

Reply:
I believe this one explicitly does not, as the SQS queue it's pulling from via the starSFWatcherRule has it's own targeted DLQ -- this allows that queue to detect if this lambda fails and puts the message in startSFFailedSQS for reprocessing.     I have a brief note on this in the resource survey on the team drive.